src/interfaces/ICSBondCurve.sol

```diff
@@ -38,13 +38,27 @@ interface ICSBondCurve {
     ///       |----------|----------|----------|----------|----> Keys Count
     ///       |          1          2          3          i
     ///
+
+    /// @dev DEPRECATED. For backward compatibility in migration only.
```

Comment:
Is that still required?

Reply:
Yep. For this lines - https://github.com/lidofinance/community-staking-module/pull/459/files#diff-b66e3d85349d83b3331aa5228e8a00d020037c26976478f55fb44fa7c116eb46R30-R31

Reply:
Changed to `bytes32`, but not sure that it makes much sense to nullify it during update